README.md

```diff
@@ -40,6 +40,7 @@ dependency can be provided two ways:
 If you do not wish to build and run the tests, add `-DBENCHMARK_ENABLE_GTEST_TESTS=OFF`
 to `CMAKE_ARGS`.
 
+For a more detailed documentation of building benchmark checkout [Additional Build Documentation](docs/Additional_Build_Documentation.md)
```

Comment:
this also covers USING benchmark.

Perhaps the docs would be clearer if we separated these two use-cases. One doc for building and installing from source, one doc for using it (linking to building from source, but also having instructions for `apt` or `vcpkg` or whatever).

Reply:
made a separate file for embedding Benchmark and moved the `vcpkg` installation to `README.md`